### PR TITLE
State

### PR DESCRIPTION
**Pull request by hlmitchell**

### CODE REVIEW

**Review by Joshua-Wright76 — APPROVED** [review]

**Review by ntsmitty — APPROVED** [review]

File: combineNodes.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // import state class for instanceof check
-const State = require('./stateNode.js');
+const StateNode = require('./stateNode.js');
 
 // ==================> FOR TESTING ONLY <=================== \\
```
Comment (G0ldenSp00n-zz) [inline combineNodes.js:4]: We Should Probably Remove this Unused Code.

Comment (G0ldenSp00n-zz) [reply combineNodes.js:4]: I would suggest moving these into a test file, and we can setup Mocha and Travis.

**Review by G0ldenSp00n-zz — CHANGES_REQUESTED** [review]
Move all the testing code into it's own testing file.

**Comment by G0ldenSp00n-zz** [comment]
Accepted Pull Request with #3 to be fixed by @Joshua-Wright76

**Review by G0ldenSp00n-zz — APPROVED** [review]

**Pull request merged**